infrasim/ipmicons/common.py

```diff
@@ -64,6 +74,12 @@ def init_env(instance):
     have a plan to give instance name to ipmi-console so that it can be attached to
     target vBMC instance.
     """
+    if not Workspace.check_workspace_exists(instance):
+        raise IpmiError("Warning: there is no node {} workspace. Please start node {} first.".format(instance, instance))
+    output = run_command("infrasim node status")
+    if not (output[0] == 0 and "stop" not in output[1]):
```

Comment:
Condition here is not rigorous. When vbmc start but qemu stop, this statement here may fail to judge real condition.